app/build.gradle

```diff
@@ -31,15 +30,17 @@ dependencies {
     implementation 'com.android.support:support-v4:26.1.0'
     implementation 'com.jakewharton.threetenabp:threetenabp:1.0.3'
     implementation 'com.google.android.gms:play-services-maps:15.0.0'
-    compile 'com.android.support:recyclerview-v7:26.1.0'
-    compile 'com.android.support:cardview-v7:26.1.0'
-    compile 'com.android.support:design:26.1.0'
-    compile 'com.android.support:appcompat-v7:26.0.1'
+    implementation 'com.android.support:recyclerview-v7:26.1.0'
+    implementation 'com.android.support:cardview-v7:26.1.0'
+    implementation 'com.android.support:design:26.1.0'
+    implementation 'com.android.support:appcompat-v7:26.0.1'
     implementation 'com.squareup.picasso:picasso:2.71828'
     implementation 'com.facebook.fresco:fresco:1.9.0'
     implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'com.apollographql.apollo:apollo-runtime:1.0.0-alpha5'
+    implementation 'com.apollographql.apollo:apollo-android-support:1.0.0-alpha5'
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
-    compile 'net.opacapp:multiline-collapsingtoolbar:1.6.0'
+    implementation 'net.opacapp:multiline-collapsingtoolbar:1.6.0'
```

Comment:
isn't it convention to have this section of the file alphabetized?

Reply:
No clue, but I'll spend some time alphabetizing it anyway.

Reply:
I'll do some research and write up a brief Android code style doc for Sunday